src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.php

```diff
@@ -0,0 +1,179 @@
+        ->set('messenger.transport.symfony_serializer', Serializer::class)
+            ->args([
+                service('serializer'),
+                '', // Format
+                [], // Context
+            ])
+        ->alias(SerializerInterface::class, 'messenger.default_serializer')
+
+        ->set('messenger.transport.native_php_serializer', PhpSerializer::class)
+
+        // Middleware
+        ->set('messenger.middleware.handle_message', HandleMessageMiddleware::class)
+            ->abstract()
+            ->args([
+                '', // Bus handler resolver
+            ])
+            ->tag('monolog.logger', ['channel' => 'messenger'])
+            ->call('setLogger', [service('logger')->ignoreOnInvalid()])
+
+        ->set('messenger.middleware.add_bus_name_stamp_middleware', AddBusNameStampMiddleware::class)
+            ->abstract()
+
+        ->set('messenger.middleware.dispatch_after_current_bus', DispatchAfterCurrentBusMiddleware::class)
+
+        ->set('messenger.middleware.validation', ValidationMiddleware::class)
+            ->args([
+                service('validator'),
+            ])
+
+        ->set('messenger.middleware.reject_redelivered_message_middleware', RejectRedeliveredMessageMiddleware::class)
+
+        ->set('messenger.middleware.failed_message_processing_middleware', FailedMessageProcessingMiddleware::class)
+
+        ->set('messenger.middleware.traceable', TraceableMiddleware::class)
+            ->abstract()
+            ->args([
+                service('debug.stopwatch'),
+            ])
+
+        // Discovery
+        ->set('messenger.receiver_locator')
+            ->args([
+                [],
+            ])
+            ->tag('container.service_locator')
+
+        // Transports
+        ->set('messenger.transport_factory', TransportFactory::class)
+            ->args([
+                tagged_iterator('messenger.transport_factory'),
+            ])
+
+        ->set('messenger.transport.amqp.factory', AmqpTransportFactory::class)
+
+        ->set('messenger.transport.redis.factory', RedisTransportFactory::class)
+
+        ->set('messenger.transport.sync.factory', SyncTransportFactory::class)
+            ->args([
+                service('messenger.routable_message_bus'),
+            ])
+            ->tag('messenger.transport_factory')
+
+        ->set('messenger.transport.in_memory.factory', InMemoryTransportFactory::class)
+            ->tag('messenger.transport_factory')
+            ->tag('kernel.reset', ['method' => 'reset'])
+
+        ->set('messenger.transport.sqs.factory', AmazonSqsTransportFactory::class)
+
+        // retry
+        ->set('messenger.retry_strategy_locator')
+            ->args([
+                [],
+            ])
+            ->tag('container.service_locator')
+
+        ->set('messenger.retry.abstract_multiplier_retry_strategy', MultiplierRetryStrategy::class)
+            ->abstract()
+            ->args([
+                '', // max retries
```

Comment:
all empty arguments with comments should use `abstract_arg`, e.g.  `abstract_arg('max retries')`